build-system/compile/single-pass.js

```diff
@@ -105,14 +105,16 @@ exports.getFlags = function(config) {
     source_map_include_content: !!argv.full_sourcemaps,
     source_map_location_mapping: ['|/'],
     //new_type_inf: true,
-    language_in: 'ES6',
     // By default closure puts all of the public exports on the global, but
     // because of the wrapper modules (to mitigate async loading of scripts)
     // that we add to the js binaries this prevents other js binaries from
     // accessing the symbol, we remedy this by attaching all public exports
     // to `_` and everything imported across modules is is accessed through `_`.
     rename_prefix_namespace: '_',
-    language_out: config.language_out || 'ES5',
+    language_in: config.esm ? 'ECMASCRIPT_2017' : 'ECMASCRIPT6',
+    language_out: config.esm
+      ? 'ECMASCRIPT_2017'
```

Comment:
Same question as above here: Should we use `NO_TRANSPILE` instead?

Reply:
`language_out` should be now `NO_TRANSPILE` when `esm` flag is used